monitoring/mongodb/dashboard.json

```
@@ -265,1465 +341,5690 @@
+          "hide": false,
           "interval": "",
-          "intervalFactor": 2,
-          "legendFormat": "",
+          "legendFormat": "Queued Writers",
+          "range": true,
+          "refId": "C"
+        }
+      ],
+      "title": "Reads & Writes clients locks",
+      "type": "timeseries"
+    },
+    {
+      "datasource": {
+        "default": true,
+        "type": "prometheus",
+        "uid": "metalk8s-prometheus"
+      },
+      "fieldConfig": {
+        "defaults": {
+          "color": {
+            "mode": "palette-classic"
+          },
+          "custom": {
+            "axisBorderShow": false,
+            "axisCenteredZero": false,
+            "axisColorMode": "text",
+            "axisLabel": "",
+            "axisPlacement": "auto",
+            "barAlignment": 0,
+            "barWidthFactor": 0.6,
+            "drawStyle": "line",
+            "fillOpacity": 19,
+            "gradientMode": "none",
+            "hideFrom": {
+              "legend": false,
+              "tooltip": false,
+              "viz": false
+            },
+            "insertNulls": false,
+            "lineInterpolation": "smooth",
+            "lineWidth": 1,
+            "pointSize": 5,
+            "scaleDistribution": {
+              "type": "linear"
+            },
+            "showPoints": "auto",
+            "spanNulls": false,
+            "stacking": {
+              "group": "A",
+              "mode": "none"
+            },
+            "thresholdsStyle": {
+              "mode": "off"
+            }
+          },
+          "mappings": [],
+          "max": 1,
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "transparent",
+                "value": null
+              }
+            ]
+          },
+          "unit": "percentunit"
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 9,
+        "w": 12,
+        "x": 0,
+        "y": 87
+      },
+      "id": 1607,
+      "options": {
+        "legend": {
+          "calcs": [],
+          "displayMode": "list",
+          "placement": "bottom",
+          "showLegend": true
+        },
+        "tooltip": {
+          "mode": "single",
+          "sort": "none"
+        }
+      },
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "metalk8s-prometheus"
+          },
+          "editorMode": "code",
+          "expr": "(\n   rate(mongodb_ss_wt_cache_pages_requested_from_the_cache{rs_state=\"1\",rs_nm=~\"$rs_nm\"}[$__rate_interval]) - \n   rate(mongodb_ss_wt_cache_pages_read_into_cache{rs_state=\"1\",rs_nm=~\"$rs_nm\"}[$__rate_interval])) / \n   rate(mongodb_ss_wt_cache_pages_requested_from_the_cache{rs_state=\"1\",rs_nm=~\"$rs_nm\"}[$__rate_interval]\n)",
+          "hide": false,
+          "instant": false,
+          "legendFormat": "Cache hit/miss ratio for {{ pod }}",
```

redundant with graph title
```suggestion
          "legendFormat": "{{ pod }}",
```